content/doc/developer/plugin-development/pipeline-integration.adoc

```diff
@@ -2,12 +2,579 @@
 layout: developerguide
 title: Writing Pipeline-Compatible Plugins
 references:
-- url: https://github.com/jenkinsci/pipeline-plugin/blob/master/DEVGUIDE.md
-  title: Plugin Developer Guide
 - url: https://github.com/jenkinsci/workflow-step-api-plugin/blob/master/README.md
```

Comment:
This is the other major component, which I suppose could be moved as a separate task.